routes/controllers/web-api.js

```diff
@@ -31,17 +49,14 @@ async function get(req, res) {
     const release = pack.versions[version];
     const latest = pack.versions[pack['dist-tags'].latest];
 
-    const maintainers = [];
-    latest.maintainers.forEach((maintainer) => {
-        maintainers.push({ name: maintainer.name, email: maintainer.email });
-    });
-
     const result = {
         name: pack.name,
         description: release.description,
         readme: release.readme,
         author: { name: release._npmUser.name, email: release._npmUser.email },
-        collaborators: maintainers,
+        collaborators: latest.maintainers.map((maintainer) => {
+            return pick(maintainer, ['name', 'email']);
+        }),
         versions: { version: Object.keys(pack.versions), time: pack.time[latest.version] },
         keywords: latest.keywords,
         dependencies: ['depA', 'depB', 'depC'],
```

Comment:
Looks like that was something I forgot to implement. Finished that for the next PR.